gotham/Cargo.toml

```diff
@@ -12,14 +12,14 @@ categories = ["web-programming::http-server"]
 keywords = ["http", "async", "web", "framework", "blockchain"]
 
 [dependencies]
-log = "0.4"
-hyper = { version = "0.11", features = [] }
+log = "0.4.1"
+hyper = { version = "0.11.7", features = [] }
```

Comment:
epic nit, ignorable.

Given we're not using any hyper features should we take the opportunity to revert back to `hyper = "0.11.7"` format?

Reply:
I thought the goal here was to explicitly opt out of hyper's default features. From looking at the latest hyper, that seems to opt us out of tokio-proto compatibility (since we don't need it). I might be misreading though.

Reply:
👍 carrying on.